README.md

```diff
@@ -113,6 +113,7 @@ Power Tuner UI repository: [mattymoomoo/aws-power-tuner-ui](https://github.com/m
 
 From most recent to oldest, with major releases in bold:
 
+* *4.2.2* (2022-10-19): configurable sleep between each function invocation
```

Comment:
Don't worry about versioning, I'll take care of this after merging the PR (not every PR becomes a new SAR version).